migrations/schemas/20230410054010-add-table-user-tags.sql

```diff
@@ -0,0 +1,15 @@
+
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS user_tags
+(
+    id UUID PRIMARY KEY NOT NULL DEFAULT uuid_generate_v4(),
```

Comment:
should we use serial (bigint or string if using snowflake) for compatible with current user id as I remember. UUID is usually used to take advantage of secure and privacy, but not good for indexing, searching, and human-readable.

Reply:
actually this table not joined with any table, and Id here just a PK. Not have any special reason, so I will change back to serial id, tks